middle/siunits.owl

```diff
@@ -165,8 +165,8 @@ email: emanuele.ghedini@unibo.it</rdfs:comment>
     <!-- http://emmo.info/emmo/middle/siunits#EMMO_1273eb34_de48_43a9_925f_104110469dd2 -->
```

Comment:
Changes in siunits.owl will most likely create many not-easy-to-resolve conflicts with PR #93. I will suggest to exclude changes to this file in this PR. PR #93 already includes the lang attribute to natural language annotations.